libs/ngx-sherfire/src/lib/auth/firebase-auth.service.ts

```diff
@@ -1,26 +0,0 @@
-import { inject, Injectable, NgZone } from '@angular/core';
-import firebase from 'firebase/app';
-import 'firebase/auth';
-import { FirebaseApp } from '../app';
-import { NgxSherfireModule } from '../ngx-sherfire.module';
-
-export interface FirebaseAuth extends firebase.auth.Auth {}
-
-@Injectable({
-    providedIn: NgxSherfireModule,
-    useFactory: firebaseAuthFactory,
-})
-export class FirebaseAuth {
-    static readonly Persistence = firebase.auth.Auth.Persistence;
```

Comment:
We removed adding all sorts of statics from "our" classes. Users of ngx-sherlock now frequently do ES imports from 'firebase/*' modules and can get all these constants there.